ffi/go/bls/callback.go

```diff
@@ -0,0 +1,12 @@
+package bls
+
+/*
+// exported from bls.go
+unsigned int wrapReadRandGo(void *buf, unsigned int n);
```

Comment:
@herumi Why do we need to interface with Go?

Reply:
DFINITY does not need the interface.
It is added for https://github.com/herumi/bls/issues/26 .

Reply:
I see 👍